cni/99-kpod-bridge.conf

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I think we already have copies of these in contrib/cni - can you either delete those, or move these there?